fb_instant_articles.info

```diff
@@ -4,3 +4,4 @@ package = Facebook Instant Articles
 core = 7.x
 php = 5.4
 configure = admin/config/services/fb-instant-articles
+dependencies[] = xautoload (>=7.x-5.x)
```

Comment:
As discussed, turns out your can get what we need out of composer_manager module, so we won't need to depend on xautoload.